### PR TITLE
Single item views

### PR DESCRIPTION
**Pull request by indirectlylit**
handling for single-item views:

* breadcrumb and table links should all work properly now
* all progress levels are calculated
* hide table for single user / single item view

### CODE REVIEW

File: kolibri/plugins/coach_tools/assets/src/vue/reports/index.vue [hunk 1]
```diff
@@ -38,65 +38,68 @@
...
+          :vlink="viewByLink"
+          :disabled="isSingleUser || isSingleItem"
+        ></view-by-switch>
+
+        <!--TABLE SECTION-->
+        <table class="data-table">
+          <thead>
+            <tr>
+              <th is="header-cell"
+                text="Name"
```
Comment (christianmemije) [inline kolibri/plugins/coach_tools/assets/src/vue/reports/index.vue]: Wrap the th strings.

**Comment by indirectlylit** [comment]
@christianmemije addressed changes - want to look & merge?

File: kolibri/plugins/coach_tools/assets/src/vue/reports/index.vue [hunk 2]
```diff
@@ -116,6 +116,13 @@
   const logging = require('kolibri.lib.logging');
 
   module.exports = {
+    $trNameSpace: 'report-page',
+    $trs: {
+      'name-col': 'Name',
```
Comment (christianmemije) [inline kolibri/plugins/coach_tools/assets/src/vue/reports/index.vue]: typo `name-col`

Comment (indirectlylit) [reply kolibri/plugins/coach_tools/assets/src/vue/reports/index.vue]: fixed

**Pull request merged**